docs/source/cloud_deployment.rst

```diff
@@ -1,11 +1,82 @@
-Benchmarking
-============
+MozDef for AWS
+===============
+
+**What is MozDef for AWS**
 
 Cloud based MozDef is an opinionated deployment of the MozDef services created in 2018 to help AWS users
 ingest cloudtrail, guardduty, and provide security services.
 
+.. image:: images/cloudformation-launch-stack.png
+   :target: https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/new?stackName=mozdef-for-aws&templateURL=https://s3-us-west-2.amazonaws.com/mozdef.infosec.allizom.org/cf/mozdef-parent.yml
+
+
+Feedback
+-----------
+
+MozDef for AWS is new an we'd love your feedback.  Try filing github issues here in the repository or connect with us
+in the Mozilla Discourse Security Category.
+
+https://discourse.mozilla.org/c/security
+
+You can also take a short survey on MozDef for AWS after you have deployed it.
+https://goo.gl/forms/JYjTYDK45d3JdnGd2
+
+
+Dependencies
+--------------
+
+MozDef requires the following:
+
+- A DNS name ( cloudymozdef.security.allizom.org )
+- An OIDC Provider with ClientID, ClientSecret, and Discovery URL
+  - Mozilla Uses Auth0 but you can use any OIDC provider you like: Shibboleth, KeyCloak, AWS Cognito, Okta, Ping (etc)
+- An ACM Certificate in the deployment region for your DNS name
+- A VPC with three public subnets available.
```

Comment:
It shouldn't need 3 public subnets, it should work with anything 2 or greater (maybe even 1)